app/assets/javascripts/bastion/components/bst-container-scroll.directive.js

```diff
@@ -34,8 +34,14 @@ angular.module('Bastion.components').directive('bstContainerScroll', ['$window',
                         offset = offset + bottomPadding;
                     }
 
-                    element.outerHeight(windowHeight - offset);
-                    element.height(windowHeight - offset);
+                    newElementHeight = windowHeight - offset;
+
+                    if (newElementHeight <= 50) {
+                        newElementHeight = 200;
```

Comment:
No, not really.   As long as 100/300 fixes the original issue I'm fine with using those instead.